openedx/core/djangoapps/programs/utils.py

```diff
@@ -327,6 +328,11 @@ def supplement_program_data(program_data, user):
         program_data (dict): Representation of a program.
         user (User): The user whose enrollments to inspect.
     """
+    for organization in program_data['organizations']:
+        org_obj = get_organization_by_short_name(organization['key'])
```

Comment:
This is a good opportunity for caching. We don't necessarily need to do it now, but I think it's worth noting.